functorch/csrc/dim/dim.cpp

```diff
@@ -1500,13 +1511,15 @@ static PyObject* _dims(PyObject *self,
     }
 
     PyThreadState* state = PyThreadState_GET();
-    PyFrameObject* f = state->frame;
-    auto code = (_Py_CODEUNIT*)PyBytes_AS_STRING(f->f_code->co_code);
+    PyFrameObject* f = PyThreadState_GetFrame(state);
```

Comment:
These can use `py::object::steal` to keep ownership in an object rather than the explicit Py_DECREF:
```
auto frame_owner = py::object::steal((PyObject*) f);
```